src/languages/sqf.js

```diff
@@ -1931,7 +2045,7 @@ export default function(hljs) {
     'setGroupIconParams',
     'setGroupIconsSelectable',
     'setGroupIconsVisible',
-    'setGroupId',
+    'setGroupid',
```

Comment:
```suggestion
    'setGroupId',
```
https://community.bistudio.com/wiki/setGroupId